README.md

```diff
@@ -47,11 +47,6 @@ Votes on a PR are sourced through the following mechanisms:
 
 ### Weights and thresholds
```

Comment:
Either this should no longer say "Weights", or the removed text below should be replaced by some text saying that all votes are weighted equally.